core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -61,6 +61,26 @@ <h3>Regenerate contribution opportunities for a topic:</h3>
   </div>
 </md-card>
 
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h3>Get Exploration Interactions</h3>
+  <div class="oppia-query-form">
+    <form class="form" ng-submit="$ctrl.retrieveExplorationInteractionIds()">
+      <div class="form-group row">
+        <label>Exploration ID:</label> <input type="text" ng-model="$ctrl.explorationId" required="true">
+      </div>
+      <div class="form-group row">
+        <input type="submit" value="submit query">
+      </div>
+    </form>
+    <p ng-if="$ctrl.explorationInteractionIds">
+      <label>Interaction IDs : </label>
+      <ul class="oppia-interaction-ids">
```

Comment:
Do we need this class-name?

Reply:
we can skip the class name but I thought it's  better to keep a name

Reply:
Considering these patterns `Don't add any unused code/service` I would suggest you remove this unused class-name. (We can re-add it whenever required! :))

Reply:
done